### PR TITLE
testing/packer: new aport

### PR DESCRIPTION
**Pull request by 7heo**

### CODE REVIEW

File: testing/packer/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+pkgname=packer
+pkgver=0.10.2
+pkgrel=0
+pkgdesc="Packer is a tool for creating identical machine images for multiple platforms from a single source configuration."
+url="https://www.packer.io/"
+arch="x86_64"
+license="MPL 2.0"
+depends=""
+depends_dev=""
+makedepends="$depends_dev go go-tools"
```
Comment (jirutka) [inline testing/packer/APKBUILD]: This package doesn’t provide -dev subpackage, so `depends_dev` should not be declared here, nor used in `makedepends`. aport tool even warns about this.

Comment (7heo) [inline testing/packer/APKBUILD]: Sorry, took the terraform APKBUILD as a template, maybe I shouldn't have.

Comment (7heo) [inline testing/packer/APKBUILD]: @jirutka Done

File: testing/packer/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# Contributor: 7heo <7heo@mail.com>
+# Maintainer: 7heo <7heo@mail.com>
+pkgname=packer
+pkgver=0.10.2
+pkgrel=0
+pkgdesc="Packer is a tool for creating identical machine images for multiple platforms from a single source configuration."
+url="https://www.packer.io/"
+arch="x86_64"
```
Comment (jirutka) [inline testing/packer/APKBUILD]: Why just x86_64, it doesn’t work on other archs?

Comment (7heo) [inline testing/packer/APKBUILD]: From the terraform APKBUILD, same as the other point.

File: testing/packer/APKBUILD [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+makedepends="go go-tools"
+install=""
+subpackages=""
+source="${pkgname}-${pkgver}.tar.gz::https://github.com/mitchellh/$pkgname/archive/v$pkgver.tar.gz"
+
+builddir="$srcdir"/$pkgname-$pkgver
+
+build() {
+	cd $srcdir
+	export GOPATH="$startdir"
```
Comment (jirutka) [inline testing/packer/APKBUILD]: I would personally prefer the approach that @gena01 used in https://github.com/alpinelinux/aports/commit/5bef15666ec397f281fa09fa57b059549902f144#diff-09aa8ce633e2619a7d2132841839fce1R19, it looks less hack-ish for me. WDYT?

**Comment by jirutka** [comment]
@7heo so what about this PR?

**Pull request closed without merging**